lib/bolt_ext/server.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'sinatra'
+require 'bolt'
+require 'json'
+
+set :server, :puma
+
+post '/ssh/run_task' do
+  body = JSON.parse(request.body.read)
+  target = [Bolt::Target.new(body['target']['hostname'], body['target']['options'])]
+  task = Bolt::Task.new(body['task']['name'], body['task']['file'], body['task']['metadata'])
+  parameters = body['parameters']
+
+  @analytics = Bolt::Analytics.build_client
+  executor = Bolt::OrchExecutor.new(Bolt::Config.new, @analytics, false)
+
+  @r = ''
```

Comment:
We probably shouldn't be setting this to empty. It seems like it can only serve to mask bugs.

We do probably need some error handling.